test/kube_testing/k8s.go

```diff
@@ -301,29 +301,29 @@ func (o *K8s) GetNodes() []v1.Node {
 }
 
 func (o *K8s) ListPods() []v1.Pod {
-	podList, err := o.clientset.CoreV1().Pods("default").List(metaV1.ListOptions{})
```

Comment:
Could we use some configurable values here?,
I suppose we could try to run few tests in parallel in case different namespaces will be used for test pod deployment. Parallel would be perfect scenario, since deployment could be not fast.

Reply:
Thanks @haiodo  This is a good proposal. I can do the multi-namespace approach in a follow up PR.